base/uk.ac.stfc.isis.ibex.jaxb/META-INF/MANIFEST.MF

```diff
@@ -8,7 +8,6 @@ Bundle-ActivationPolicy: lazy
 Export-Package: uk.ac.stfc.isis.ibex.jaxb
 Bundle-ClassPath: .
 Require-Bundle: jakarta.xml.bind-api;bundle-version="2.3.2";visibility:=reexport,
- com.sun.xml.bind.jaxb-osgi;bundle-version="2.3.2",
```

Comment:
There is a compile error in `opis.tests` due to this dependency being removed. It seems that also removing the dependency from `opis.tests` works from eclipse.